terraform/gcp/variables.tf

```diff
@@ -775,3 +775,29 @@ variable "iscsi_remote_python" {
   default     = "/usr/bin/python3"
 }
 
+# Peering related variables
+
+variable "enable_ibsm_peering" {
+  description = "Enable VPC peering between HANA VPC and ibsm VPC"
+  type        = bool
```

Comment:
it can be `int` with default to zero. Doing so we can also simplify all the 
```
count = var.enable_ibsm_peering ? 1 : 0
```

to 
```
count = var.enable_ibsm_peering
```